AzureStack/2015-04-01/swagger/GalleryAdmin.json

```diff
@@ -0,0 +1,502 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "GalleryAdminClient",
+    "description": "Management client for azurestack gallery admin operations.",
+    "version": "2015-04-01"
+  },
+  "host": "publicapi.azurestack.local",
+  "schemes": [
+    "https"
+  ],
+    "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+      "/subscriptions/{subscriptionId}/providers/microsoft.gallery.admin/galleryItems": {
+      "get": {
+        "tags": [
+          "AdminGalleryItems"
+        ],
+        "description": "Lists gallery registrations.",
+        "operationId": "AdminGalleryItems_List",
```

Comment:
Is this a pageable operation? Does the server return more than one server page in the response? 
- If yes, then 
   - `"x-ms-pageable": { "nextLinkName": "nextLink" }`
- If no, then 
  -  `"x-ms-pageable": { "nextLinkName": null }`